modules/bullet/rigid_body_bullet.cpp

```diff
@@ -535,20 +535,21 @@ void RigidBodyBullet::set_mode(PhysicsServer::BodyMode p_mode) {
 			reload_axis_lock();
 			_internal_set_mass(0);
 			break;
-		case PhysicsServer::BODY_MODE_RIGID: {
+		case PhysicsServer::BODY_MODE_RIGID:
 			mode = PhysicsServer::BODY_MODE_RIGID;
 			reload_axis_lock();
 			_internal_set_mass(0 == mass ? 1 : mass);
 			scratch_space_override_modificator();
 			break;
-		}
-		case PhysicsServer::BODY_MODE_CHARACTER: {
+		case PhysicsServer::BODY_MODE_CHARACTER:
 			mode = PhysicsServer::BODY_MODE_CHARACTER;
 			reload_axis_lock();
 			_internal_set_mass(0 == mass ? 1 : mass);
 			scratch_space_override_modificator();
 			break;
-		}
+		case PhysicsServer::BODY_MODE_SOFT:
+			// FIXME: Implementation needed?
+			break;
```

Comment:
```
modules/bullet/rigid_body_bullet.cpp:526:9: warning: enumeration value 'BODY_MODE_SOFT' not handled in switch [-Wswitch]
```
@AndreaCatania

Reply:
It's a rigid body, probably a warning or error message could be set inside instead of the comment.

Reply:
You mean that `BODY_MODE_SOFT` is not relevant as `mode` value for `RigidBody`, right? In the editor it doesn't seem to be exposed as an option indeed.

If so, what is `BODY_MODE_SOFT` used for? It doesn't seem to be used anywhere prior to this commit.

Reply:
See https://github.com/godotengine/godot/pull/22447/commits/a9957233e842e1863f4d6fd181b55351013f2b2a#diff-cf5f9c8a55addaf91ea1f36f7234ff9eL363 - as far as I can tell we don't need this enum value anywhere, so it's best to remove it while we can.